README.md

```diff
@@ -65,11 +65,23 @@ In order to run or edit the examples:
 
 1. Assert an existing Exabyte.io account. Examples require an account to run. New users can register [here](https://platform.exabyte.io/register) to obtain one.
 
-2. Open [settings](examples/settings.py) and adjust it to provide the API authentication parameters. See the [corresponding example](examples/system/get_authentication_params.ipynb) to learn how to obtain the authentication parameters.
+2. Open [settings](examples/settings.json) and adjust it to provide the API authentication parameters. See the [corresponding example](examples/system/get_authentication_params.ipynb) to learn how to obtain the authentication parameters.
 
 3. Open the desired example notebook, adjust it as necessary and run.
 
 
+## Running Notebooks
+
+1. For running notebooks, one follows a similar approach mentioned in the 'Setup' and 'Usage' sections. After the user has created their virtual environment (step3 in the 'Setup' section), navigate to the `examples/` folder and launch the Jupyter Notebook environment. These steps are shown below. Note: In order for the post-save hook feature to work properly, one must launch their Jupyter Notebook environment in the folder that contains the file `config.py`.
```

Comment:
(1) We should be careful and not intermix the Notebook and Lab, and the command has `lab` but you are referring to Notebook. Let's explain or just use Lab.

(2) Let's adjust step 4 in the "Setup" section above to refer to "Running Notebooks" or just remove it to avoid duplication

Reply:
Thanks. How about I place a note after step 3 in the 'Setup' section?